x-pack/metricbeat/module/aws/utils_test.go

```diff
@@ -238,7 +238,7 @@ func TestGetMetricDataPerRegion(t *testing.T) {
 	reqGetMetricData := mockSvc.GetMetricDataRequest(getMetricDataInput)
 	getMetricDataOutput, err := reqGetMetricData.Send(context.TODO())
 	if err != nil {
-		fmt.Println("failed getMetricDataPerRegion: ", err)
+		_ = fmt.Errorf("failed getMetricDataPerRegion: %w", err)
```

Comment:
nit: maybe we can also use `assert.NoError(t, err)` here instead.

Reply:
ah thanks for this - i was blindly fixing linter errors and didn't stop to actually think about it.